packages/manager/apps/telecom/src/app/telecom/telephony/svaWallet/kyc/identity/overview-identity/overview-identity.html

```diff
@@ -0,0 +1,340 @@
+<div data-ng-if="!$ctrl.editMode">
+    <h3
+        data-translate="telephony_billingAccount_svaWallet_kyc_identity_title"
+    ></h3>
+
+    <div class="oui-field oui-field_auto">
+        <div class="oui-field__control">
```

Comment:
Is there a reason for using `oui-field` classes where no fields chlidren element are set? Correct me I'm wrong but it looks like that a definition list could fit what you need

Reply:
We have wireframes to respect between creation and reading the information.
oui-tile break wireframes.